posthog/temporal/data_imports/pipelines/pipeline_sync.py

```diff
@@ -194,8 +194,14 @@ def _run(self) -> dict[str, int]:
                 delta_uri = f"{settings.BUCKET_URL}/{self.inputs.dataset_name}/{name}"
                 storage_options = self._get_credentials()
 
-                if DeltaTable.is_deltatable(delta_uri, storage_options):
-                    delta_table = DeltaTable(delta_uri, storage_options=self._get_credentials())
+                self.logger.debug(f"delta_uri={delta_uri}")
```

Comment:
just FYI you can also do `f"{delta_uri=}"

Reply:
🤯